hack/update-all.sh

```diff
@@ -54,7 +54,7 @@ if ! $ALL ; then
 	echo "Running in short-circuit mode; run with -a to force all scripts to run."
 fi
 
-kube::util::ensure_godep_version v74
+kube::util::ensure_godep_version v79
```

Comment:
I had the discussion with @DirectXMan12 somewhere already. If we have a global constant, it will happen that you change it to fix one of the scripts, and by doing this you break 3 other scripts you don't test. Happened to me when introducing kube::util::ensure_godep_version in the first place. Hence, I prefer to have explicit versions per scripts.